### PR TITLE
add KubeCon 2024 Paris

### PR DESCRIPTION
**Pull request by dirien**
add KubeCon 2024 Paris

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,3 +279,7 @@ All the CFPs (past and coming) are available publicly in JSON: https://developer
 
 * 17-19: [Devoxx France](https://www.devoxx.fr/) - Paris (France)
 * 25-26: [MiXiT](https://mixitconf.org/fr/) - Villeurbanne (Lyon France)
+
+### March
+
+* 19-22: [KubeCon + CloudNativeCon Europe 2024](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe-2024/) - Paris (France)
```
Comment (scraly) [inline README.md:285]: Thanks!
Just a remark, is it possible to copy paste this new March section before April? 😇
Thx🥰

**Review by scraly — CHANGES_REQUESTED** [review]
Thank you
Just one remark in order to have the list of events ordered by month ☺️

File: README.md (hunk 1 above)
Comment (dirien) [reply README.md]: Done! I should learn the months 😅

**Comment by dirien** [comment]
@scraly thanks for the feedback! I should learn the months! 😄

**Comment by scraly** [comment]
Ah ah thanks @dirien for your contribution 🥰
@davidaparicio you know I am like a fox, I am trying to see everything... Even in my small phone 😆😆

**Pull request merged**